src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -743,36 +744,10 @@ private static void handleHeadlessGameServerArgs() {
           + (2 * GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT) + " seconds.");
       printUsage = true;
     }
-    // no passwords allowed for bots
-    // take any actions or commit to preferences
-    final String clientWait = System.getProperty(GameRunner.TRIPLEA_SERVER_START_GAME_SYNC_WAIT_TIME, "");
```

Comment:
A lot of this was more complicated than need be. We already have a very slow timeout, and we adjust it by less than 5%. I removed most of this since the adjustments are not that significant. Most of it was outright removed as our `ClientSetting` infrastructure supports it pretty directly.